src/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { PathExt } from '@jupyterlab/coreutils';
+import { IFileBrowserFactory } from '@jupyterlab/filebrowser';
+import { GithubPuller } from './githubpuller';
+
+const assignmentListExtension: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlite/litegitpuller.plugin',
```

Comment:
Maybe using a `:` for consistency?

```suggestion
  id: '@jupyterlite/litegitpuller:plugin',
```